app/src/main/java/com/github/libretube/util/DashHelper.kt

```diff
@@ -0,0 +1,176 @@
+package com.github.libretube.util
+
+import com.github.libretube.api.obj.PipedStream
+import com.github.libretube.api.obj.Streams
+import org.w3c.dom.Document
+import org.w3c.dom.Element
+import java.io.StringWriter
+import javax.xml.parsers.DocumentBuilder
+import javax.xml.parsers.DocumentBuilderFactory
+import javax.xml.transform.TransformerFactory
+import javax.xml.transform.dom.DOMSource
+import javax.xml.transform.stream.StreamResult
+
+// Based off of https://github.com/TeamPiped/Piped/blob/master/src/utils/DashUtils.js
+
+class DashHelper {
```

Comment:
Maybe we could just use the Kotlin in-built `object` here instead of a class with a companion object? :)

Reply:
I have no clue how Kotlin works haha!

Reply:
Just Java but in easier :p